### ISSUE TITLE
@ValueSource explanation in user guide is misleading

### ISSUE THREAD

**Issue by oohira** [junit-team/junit5#1901]
I think the explation of `Null and Empty Sources` section is a little misleading. In the user guide, 

* https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-null-and-empty

> If you need to supply multiple varying types of empty or blank strings to a parameterized test, you can achieve that using `@ValueSource` — for example, `@ValueSource(strings = {" ", " ", "\t", "\n"})`.

I think 

1. `@ValueSource(strings = {" ", " ", "\t", "\n"})` is typo of `@ValueSource(strings = {" ", "   ", "\t", "\n"})` (NOTE: the 1st argument is 1x space, and the 2nd argument is 3x space. GitHub does not support to display it ...?) because the 1st and 2nd arguments are same, and inconsistent with the folowing code example.

    ```java
    @ParameterizedTest
    @NullSource
    @EmptySource
    @ValueSource(strings = { " ", "   ", "\t", "\n" })
    void nullEmptyAndBlankStrings(String text) {
        assertTrue(text == null || text.trim().isEmpty());
     }
    ```
1. explanation of `supply multiple varying types of empty or blank strings ...` is misleading because the above example of `@ValueSource` produces four parameters all of which are blank string but not empty string.
    * To produce an empty string, you should use `@ValueSource` with `@EmptySource` or specified `strings = {"", ...}` args for `@ValueSource`.

**Comment by sbrannen** [comment]
Thank you for bringing this to our attention.

I have addressed both of these issues in a3d0f53a0bc61cd1695b25465a6114b50535e1ce.

**Comment by oohira** [comment]
Thanks for your quick fix!
